sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -116,7 +116,7 @@ Authentication with AAD requires some initial setup:
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-identity</artifactId>
-    <version>1.0.6</version>
+    <version>1.0.8</version>
```

Comment:
@JimSuplizio  I think the version is not updated in readme after Identity released.

https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/textanalytics/azure-ai-textanalytics#create-a-text-analytics-client-with-azure-active-directory-credential

Reply:
Adding @g2vinay. With GA identity being released out of a feature branch and the dependency version being updated by hand after that release, it looks like this may have just been missed.